xslt/Instrument.xsl

```diff
@@ -72,28 +74,102 @@ ome/ome.xsd: 979 # # This means that for more details on how this section of the
     <!-- zisraw/Instrument.xsd: 50 -->
     <!-- ome/ome.xsd: 2039 -->
     <xsl:template match="Microscope">
-        <MicroscopeBody>
+        <xsl:element name="ome:Microscope">
+            <!-- Not in OME but in BINA -->
+            <!-- <xsl:apply-templates select="@Id"/> -->
+            <!-- <xsl:apply-templates select="@Name"/> -->
+            <!-- <xsl:apply-templates select="Type"/> -->
```

Comment:
Normally I would say remove but this is a good enough comment for me to say keep it. BINA / 4DN stuff we can pull in later.